src/base/bittorrent/session.cpp

```diff
@@ -3068,6 +3070,8 @@ void Session::startUpTorrents()
         if (!addTorrent_impl(params.addTorrentData, params.magnetUri, TorrentInfo::loadFromFile(filePath), params.data))
             logger->addMessage(tr("Unable to resume torrent '%1'.", "e.g: Unable to resume torrent 'hash'.")
                                .arg(params.hash), Log::CRITICAL);
+        // don't waste memory with to many unprocessed queue entries
+        readAlerts();
```

Comment:
Would be a little bit more work because the correct code style doesn't allow much. I need to rebild the logic for ordered add which is correnlty done by several counters and a tmp storage.